### PR TITLE
week1 - homework - Zekiye Cakiral

### PR DESCRIPTION
**Pull request by zekiyecakiral**

### CODE REVIEW

File: Week1/homework/js-exercises/Exercise1.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+console.log("Merhaba, dunya!");   // Turkish
+console.log("Hello world");       // English
+console.log("Halo, dunia!");      // Indonesian
+console.log("Ciao, mondo!");      // Italian
+console.log("Hola, mundo!");      // Spanish
+console.log("Bonjour, monde!");   // French
+console.log("Hallo, wereld!");    // Dutch
+console.log("Hallo, welt!");      // German
+console.log("Hallo, verden!");    // Norwegian
+console.log("Hei, maailma!");     // Finnish
```
Comment (burakcoskun) [inline Week1/homework/js-exercises/Exercise1.js:11]: nice

File: Week1/homework/js-exercises/Exercise10.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+const array2 = ["Hello", 123, true, { name: "Zekiye"},{surname : "Cakiral"},29,"Istanbul"];
+
+console.log('The length of the array2 is ' + array2.length);
+
+if(array1.length == array2.length){
+    console.log("They are the same!");
+}else{
+    console.log("Two different sizes");
+}
```
Comment (burakcoskun) [inline Week1/homework/js-exercises/Exercise10.js:15]: looks good to me

File: Week1/homework/js-exercises/Exercise2.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+"use strict";
+// console.log('I'm awesome');
+console.log("'I'm awesome");
```
Comment (burakcoskun) [inline Week1/homework/js-exercises/Exercise2.js:3]: nice

File: Week1/homework/js-exercises/Exercise3.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+"use strict";
+let  numberX;
+console.log("I think the value of X is undefined");
+console.log(numberX);
+numberX = 15;
+console.log("I think the value of X is 15");
+console.log(numberX);
```
Comment (burakcoskun) [inline Week1/homework/js-exercises/Exercise3.js:7]: nice

File: Week1/homework/js-exercises/Exercise4.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+let myString = "Zekiye Cakiral";
+console.log("I think the value of myString is 'Zekiye Cakiral'");
+console.log(myString);
+
+myString = "Burak Kilinc";
+console.log("I think the value of myString is 'Burak Kilinc'");
+console.log(myString);
```
Comment (burakcoskun) [inline Week1/homework/js-exercises/Exercise4.js:9]: good

File: Week1/homework/js-exercises/Exercise5.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+let z = 7.25;
+console.log("The value of z is "+ z);
+
+let a = Math.round(z);
+console.log("The value of a is "+ a);
+
+let b = Math.max(z,a);
+
```
Comment (burakcoskun) [inline Week1/homework/js-exercises/Exercise5.js:10]: good

File: Week1/homework/js-exercises/Exercise6.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+let names=[];
+console.log("This array is empty");
+console.log(names);
+
+let animals =["cat","dog","bird"];
+console.log(animals);
+
+animals.push("Piglet");
+console.log(animals);
```
Comment (burakcoskun) [inline Week1/homework/js-exercises/Exercise6.js:11]: looks good to me

File: Week1/homework/js-exercises/Exercise7.js [hunk 8]
```diff
@@ -0,0 +1,4 @@
+"use strict";
+
+let mySentence ="Programming is so interesting!";
+console.log("Length is "+mySentence.length);
```
Comment (burakcoskun) [inline Week1/homework/js-exercises/Exercise7.js:4]: nice

File: Week1/homework/js-exercises/Exercise8.js [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+if(typeof y ===  typeof student1){
+    console.log('SAME TYPE');
+}else{
+    console.log('NOT THE SAME');
+}
+if(typeof y ===  typeof student2){
+    console.log('SAME TYPE');
+}else{
+    console.log('NOT THE SAME');
+}
```
Comment (burakcoskun) [inline Week1/homework/js-exercises/Exercise8.js:48]: nice that you compared everything with everything else

File: Week1/homework/js-exercises/Exercise9.js [hunk 10]
```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+let x = 7;
+let y = 21;
+let z = 13;
+
+console.log("The remainder operator % gives the remainder of the division of two numbers");
+if(x == 7){
```
Comment (burakcoskun) [inline Week1/homework/js-exercises/Exercise9.js:8]: don't need if conditions here because x is clearly 7

**Review by burakcoskun — APPROVED** [review]
good homework thanks

**Comment by zekiyecakiral** [comment]
> good homework thanks

thanks for your feedback

**Pull request closed without merging**